decidim-admin/app/helpers/decidim/admin/paginable/per_page_helper.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Admin
+    module Paginable
+      # This module includes helpers the :per_page cell's option
+      module PerPageHelper
+        def per_page_options
```

Comment:
If this method does not need to make use of Action View helpers, I would rather move this logic inside the `Decidim::Admin::Paginable` concern.

Instead, it would be nice to expose some helper method to easily render the cell from the views, i.e.:
https://github.com/decidim/decidim/blob/469da35bec037e17671c1923780f0ca1a4e39b60/decidim-core/app/helpers/decidim/admin/filters_helper.rb#L48